wp-content/plugins/ig-revisions/plugin.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Plugin Name: Integreat Page Revisions
+ * Description: Set a specific page revision for delivery via API
+ * Version: 1.0
+ * Author: Integreat Team / Sven Seeberg
+ * Author URI: https://github.com/Integreat
+ * License: MIT
+ * Text Domain: ig-revisions
+ */
+
+function ig_revisions_metabox( $post ) {
+	$children = wp_get_post_revisions( $_GET['post'] );
+	$revision_id = get_post_meta( $_GET['post'], 'ig_revision_id', true );
+	$options = "<option value='-1'" . (!$revision_id ? " selected" : "") . ">Current</option>";
```

Comment:
Ok, but as the meta box will be primarily used by non-tecs, they might find it confusing if there are two options in a select tag which have basically the same meaning, but different labels. E.g., if there are 3 revisions, the select tag would now look like this:
```
Current
$revision_3->post_date
$revision_2->post_date
$revision_1->post_date
```
In my opinion, something like the following would be more intuitive:
```
$revision_3->post_date (Current)
$revision_2->post_date
$revision_1->post_date
```
Or we could also omit the "Current" completely, because users might be confused about whether it refers to the current state in the editor or the current state in the API.

Reply:
No, we cannot omit it. There must be one option to deliver the most current version then publishing a post. Otherwise users would have to manually set the newest version all the time. In most cases they don't want to use revisions. We can rename it to "Always deliver newest version"

Reply:
Oh sorry, I completely misunderstood the purpose of this field. Nevermind, the option is fine 😅